examples/c++-api/CRUDExample.cc

```diff
@@ -2,9 +2,9 @@
 
 #include <opencog/atomspace/AtomSpace.h>
 #include <opencog/attentionbank/AttentionBank.h>
-#include <opencog/truthvalue/AttentionValue.h>
-#include <opencog/truthvalue/SimpleTruthValue.h>
-#include <opencog/truthvalue/TruthValue.h>
+#include <opencog/atoms/truthvalue/AttentionValue.h>
```

Comment:
I just realized that `AttentionValue.h` was under `truthvalue` but it has nothing to do here, right? @linas? Shouldn't it moved in its own directory? Anyway, I don't want to add confusion to that PR, we can really discuss that in another PR.

Reply:
Don't care. We could leave it there, or move it to the `attentionbank` directory (thereby indicating that one should never use it, unless one is also using the attentionbank.)